Examples/WhisperAX/WhisperAX/Views/ContentView.swift

```diff
@@ -57,7 +57,7 @@ struct ContentView: View {
     @State private var currentFallbacks: Int = 0
     @State private var lastBufferSize: Int = 0
     @State private var lastConfirmedSegmentEndSeconds: Float = 0
-    @State private var requiredSegmentsForConfirmation: Int = 2
+    @State private var requiredSegmentsForConfirmation: Int = 4
```

Comment:
Updating this to account for #24 increasing the amount of segments generated overall